pypeman/endpoints.py

```diff
@@ -61,6 +65,7 @@ def mk_socket(sock, reuse_port):
             if not return sock as is
         """
         if isinstance(sock, str):
+            logger.debug("trying to create socket %s", sock)
```

Comment:
`logger.debug("trying to create socket %s", repr(sock))`

Reply:
already a string